Codeherder19/redactor
# Feature/testing and refactoring

### What does this PR do?
- This PR sets up tests via Cypress. The application is tested to make sure the necessary elements appear where they should, and different strings are used in the input and text area, verifying the expected output when the submit button is clicked. 

### What does it fix?
- No bugs to fix at this point.

### Is this a feature or a fix?
- This is a feature, doing some end to end testing to ensure I receive the correct output when given a certain input.

### Where should the reviewer start?
- Clone down this repo to your machine. Change into the directory and run the following commands:

### How should this be tested?
- Change into the directory and run the following commands:
- Type npm start in your terminal to open the application.
- Type npm run cypress in your terminal to verify all tests are passing.
